ShellPkg/Application/Shell/ConsoleLogger.c

```diff
@@ -719,13 +719,27 @@ ConsoleLoggerPrintWithPageBreak (
   CONST CHAR16  *LineStart;
   CHAR16        *StringCopy;
   CHAR16        TempChar;
+  INT32         curCol;    // MU_CHANGE

```

Comment:
I see the commit message here is Drop and add to a new PR, I assume you intend to do that?

Reply:
yup plan is to drop them and add them as a seperate smaller PR